magento/devdocs#: Add sendEmailToFriend mutation API errors (#7124)
magento/devdocs: src/guides/v2.3/graphql/mutations/send-email-to-friend.md

`email` | String | The email address of the sender
`message` | String | The text of the message
`name` | String | The name of the sender

## Errors

> 

Reviewer: Listing the click paths seems overly specific in this section. Add a line similar to the following:

`Some errors occur because the **Email a friend** feature is not fully enabled. Go to **Stores** > **Configuration** > **Catalog** > **Email to a friend** in the Admin to adjust the settings.`

Author: **done**

Author: **done**

Error | Description
--- | ---

> `"Email to a Friend" is not enabled.` | "Email to a Friend" is disabled. To activate it the `Stores` > `Configuration` > `Catalog` > `Email to a friend` > `Enabled` field must be set to `Yes`.

Reviewer: ```suggestion
`"Email to a Friend" is not enabled.` | "Email to a Friend" is disabled. To activate it, use the Admin to set the **Enabled** field to **Yes**.
```

Author: **done**

`Please provide Name of sender.` | The value specified in the `input`.`sender`.`name` argument is empty.
`Please provide Email of sender.` | The value specified in the `input`.`sender`.`email` argument is empty.
`Please provide Message.` | The value specified in the `input`.`sender`.`message` argument is empty.
`Please provide Name for all of recipients.` | The value specified in the `input`.`recipients`[].`name` argument is empty.
`Please provide Email for all of recipients.` | The value specified in the `input`.`recipients`[].`email` argument is empty.

> `The current customer isn't authorized.` | "Email to a Friend" is available for a registered users only. To make it available for guests set the `Stores` > `Configuration` > `Catalog` > `Email to a friend` > `Allow for Guests` option to `Yes`.

Reviewer: ```suggestion
`The current customer isn't authorized.` | "Email to a Friend" is available for registered users only. To make it available for guests, use the Admin to set the  **Allow for Guests** option to **Yes**.
```

Author: **done**

> `The product that was requested doesn't exist. Verify the product and try again.` | Product specified in the `product_id` argument is not visible in a current website.

Reviewer: ```suggestion
`The product that was requested doesn't exist. Verify the product and try again.` | The product specified in the `product_id` argument is not visible in the current website.
```

Author: **done**

> `You can't send messages more than XXX times an hour.` | User can not send count of messages per an hour more than specified in the `Stores` > `Configuration` > `Catalog` > `Email to a friend` > `Max Products Sent in 1 Hour` config option.

Reviewer: ```suggestion
`You can't send messages more than XXX times an hour.` | The user cannot send more messages in an hour than specified in the  **Max Products Sent in 1 Hour** option in the Admin.
```

Author: **done**

> `You can't send messages more than XXX times an hour.` | User can not send count of messages per an hour more than specified in the `Stores` > `Configuration` > `Catalog` > `Email to a friend` > `Max Products Sent in 1 Hour` config option.

Reviewer: ```suggestion
`You can't send messages more than XXX times an hour.` | The user cannot send more messages in an hour than specified in the `Max Products Sent in 1 Hour` configuration option.
```

Author: **done**
